File: Data Structures & Algorithms/valid-binary-search-tree/submission-2.java
/**
 * Definition for a binary tree node.
 * public class TreeNode {
 *     int val;
 *     TreeNode left;
 *     TreeNode right;
 *     TreeNode() {}
 *     TreeNode(int val) { this.val = val; }
 *     TreeNode(int val, TreeNode left, TreeNode right) {
 *         this.val = val;
 *         this.left = left;
 *         this.right = right;
 *     }
 * }
 */

class Solution {
    public boolean isValidBST(TreeNode root) {
        if(root == null)
        {
            return true;
        }
        boolean ans = true;
        if(root.left!=null)
        {
            ans = ans && (root.left.val<root.val);

        }  
        if(root.right!=null)
        {
            ans = ans && (root.right.val>root.val);

        } 
        return ans && isValidBST(root.left) && isValidBST(root.right);

    }
}

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-2.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        ans=[]
        if root==None:
            return ans
        q=deque()
        q.append(root)
        while len(q)>0:
            l= len(q)
            row=[]
            for i in range(l):
                r= q.popleft()
                if r.left:
                    q.append(r.left)
                if r.right:
                    q.append(r.right)
                row.append(r.val)
            ans.append(row)

        return ans